wiki/FitNesseRoot/HsacAcceptanceTests/SlimTests/BrowserTest/ValueOfIsValid.wiki

```diff
@@ -0,0 +1,59 @@
+This test ensures we can check if an entered value is valid, if a pattern attribute is specified to do the validation.
```

Comment:
Please start file with
```
---
Test
---
```

To ensure wiki sees it as test page

Reply:
Ok